### PR TITLE
Music player and music queue view

### PR DESCRIPTION
**Pull request by pfernandez01**

### CODE REVIEW

File: www/app/locals/local/service.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  VenuePlayerService.$inject = ['$rootScope', 'env', '$firebaseArray'];
+
+  /* @ngInject */
+  function VenuePlayerService($rootScope, env, $firebaseArray) {
+    this.fetchQueue = fetchQueue;
+    this.onYouTubeIframeAPIReady = onYouTubeIframeAPIReady;
+
+    var ref = new Firebase(env.firebaseApiUrl);
+    var uid = $rootScope.authenticatedUser.uid;
+    var upcomingSongs = [
```
Comment (Emmurillo) [inline www/app/locals/local/service.js]: This should be handled on the controller with the service's help, the same for the ended event logic.

File: www/app/locals/local/service.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    }
+
+    function onPlayerStateChange(event) {
+      if (event.data == YT.PlayerState.ENDED) {
+        actualSong = actualSong + 1;
+        launchNextSong(upcomingSongs[actualSong].id, upcomingSongs[actualSong].title);
+      }
+    }
+
+    function launchNextSong(id, title) {
```
Comment (Emmurillo) [inline www/app/locals/local/service.js]: Same for this one.

File: www/app/locals/local/controller.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  .controller('VenuePlayerCtrl', VenuePlayerCtrl);
+
+  VenuePlayerCtrl.$inject = ['$stateParams', 'VenuePlayerService'];
+
+  /* @ngInject */
+  function VenuePlayerCtrl($stateParams, VenuePlayerService) {
+    var vm = this;
+    vm.localID = $stateParams.localID;
+    vm.queue = [];
+    vm.playVideo = playVideo;
```
Comment (emmurillo) [inline www/app/locals/local/controller.js]: I guess the right name for this would say `playQueue`

File: www/app/locals/local/view.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        <p>Nine Inch Nails</p>
+      </div>
+
+      <div class="item item-image video-container">
+        <div id="player"></div>
+      </div>
+
+      <a class="item item-icon-left assertive" href="#" ng-click="vm.playVideo()">
+        <i class="icon ion-music-note"></i>
+        Play Video
```
Comment (emmurillo) [inline www/app/locals/local/view.html]: `Play Queue`

**Pull request merged**